qcengine/programs/turbomole/define.py

```diff
@@ -0,0 +1,209 @@
+from subprocess import Popen, PIPE, TimeoutExpired
+from typing import Any, Dict, Optional
+
+from .methods import METHODS, KEYWORDS
+
+
+def execute_define(stdin: str, cwd: Optional["Path"] = None) -> str:
+    # TODO: replace this with a call to the default execute provided by QCEngine
+    # if possible. May be difficult though, as we have to pipe in stdin and
+    # be careful with the encoding.
+
+    # We cant use univeral_newlines=True or text=True in Popen as some of the
+    # data that define returns isn't proper UTF-8, so the decoding will crash.
+    # We will decode it later on manually.
+    proc = Popen("define",
+                 stdin=PIPE, stdout=PIPE, stderr=PIPE, cwd=cwd,
+                 # stdin=PIPE, stderr=PIPE, cwd=cwd,
+    )
+    # TODO: Add timeout? Unless the disk hangs this should never take long...
+    # TODO: If used with timeout an exception will be thrown. Then how do we
```

Comment:
This data should be contained within the `error.error_message` field. You can also simply raise a message like [here](https://github.com/MolSSI/QCEngine/blob/master/qcengine/programs/psi4.py#L134) and everything will be handled automatically.

Reply:
(hopefully?) fixed in 12854d6. I am not completly if i did it correctly ...